### DISCUSSION TITLE
What is the status of v5?

### DISCUSSION THREAD

**Discussion by Romfos** [AutoFixture/AutoFixture#1298] [Q&A] [answered]
Hi,

Is AutoFixture dead?
As I see the latest commit in v5 was more than year ago

Thank you

**Comment by aivascu** [comment, marked as answer]
### Update 2025:  the v5 status is on #1327 as for the slow updates see #1469
--- 
Hello @Romfos 

TL;DR No it is not dead.

There is currently work being done outside v5, and I am maintaining an active presence on StackOverflow and in GitHub Discussions, for anyone who needs help.
 
There were some issues with configuring the CI for the v5 branch. Those issues remained unresolved till last month when I have found a way to unblock things.

One issue was with making versioning work based on the branch name. I tried (many) different implementations, none of which worked properly. In the end I decided to move to GitVersion. In CD mode it supports versioning using release branches. So expect the `v5` branch to be renamed to `release/5.0.0` in the following days.

Another issue is AppVeyor throwing a null reference exception when uploading test results files without records. This issue appeared after moving the build script to NUKE. I have been investigating this for a long time. The solution I believe will be to ignore the error since it doesn't affect anyway the number of tests being displayed in the summary.

In the near future I think the project will move away from AppVeyor. Likely the replacement will be GitHub Actions.

Admittedly I have not been as active as I would have liked. That is due to my job and personal life, taking most of my time. It is really hard to find motivation, to do anything, after a day dealing with legacy code, and a dozen meetings.

Please keep in mind that AutoFixture is a community driven project, and we rely on contributions from the community to keep growing. Leave a star on the repository, so this project can find more contributors/maintainers.

**Comment by sensslen** [comment]
Is there a timeline for the v5 release? As I understand NUnit 4 support will only come in v5. NUnit 4 has now been out for almost a year. Autofixture is currently holding back quite a few nuget updates on my side. It would be great to at least get some rough timeframe on when v5 is planned.

**Reply by aivascu** [reply]
@sensslen, no, there currently is no timeline. I tried doing that, but it kept failing because other things took priority. Right now, I'm working whenever I have spare time or when I can schedule vacation days.
I am currently reprioritizing some stuff planned for v4 & v5. I would really like to release v5 before the .NET 9 release in November, but it remains to be seen whether I'll have the time.

**Reply by rbev** [reply]
we ended up just using the 5.0 preview builds and whitelisting it from our prerelease checks across our build system - it's the only package we have whitelisted but so far haven't seen any issues.

**Comment by evilbaschdi** [comment]
any update on that?

**Reply by rbev** [reply]
project seems dead

**Reply by aivascu** [reply]
@evilbaschdi the v5 status is on #1327 as for the slow updates see #1469.
